tmt/steps/execute/scripts/tmt-reboot

```diff
@@ -18,6 +18,7 @@ while getopts "c:t:e" flag; do
         c) command="${OPTARG}";;
         t) timeout="${OPTARG}";;
         e) efi=False;;
+        *) exit 1;;
```

Comment:
You may want to put a message there to the user about invalid argument.

Reply:
This is already handled by `getopt`:

    tmt-reboot: illegal option -- x

So I guess we don't have to duplicate the error message.